styles/index_style.css

```diff
@@ -0,0 +1,94 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: sans-serif;
+}
+
+body {
+    position: relative;
+    width: 100%;
+    height: 100vh;
+    background: #EFE2BA;
+}
+
+header {
+    display: flex;
+    width: 84%;
+    margin: auto;
+    padding: 20px 0;
+    align-items: center;
+    justify-content: space-between;
+}
+
+nav ul li {
+    display: inline-block;
+    list-style: none;
+    margin: 10px 20px;
+}
+
+nav ul li a {
+    text-decoration: none;
+    color: #4056A1;
+    font-weight: bold;
+}
+
+nav ul li a:hover {
+    color: #F13020;
+}
+
+.intro {
+    margin-left: 8%;
+    margin-top: 13%;
+}
+
+.intro h1 {
+    font-size: 50px;
+    color: #4056A1;
+    margin-bottom: 20px;
+}
+
+span {
+    color: #D79922;
+}
+
+.intro p {
+    color: #555;
+    line-height: 22px;
+}
+
+.intro a {
+    background: #4056A1;
+    padding: 10px 18px;
+    text-decoration: none;
+    font-weight: bold;
+    color: #D79922;
+    display: inline-block;
+    margin: 30px 0;
+    border-radius: 5px;
+}
+
+#images {
+    width: 45%;
+    height: 80%;
+    position: absolute;
+    bottom: 0;
+    right: 100px;
+}
```

Comment:
This looks great on a big screen, but the image starts to overlap the text on a smaller screen. Not a big deal for this project! But as an optional exercise can you consider how you might make it so your site resizes a bit better?